test/libsolidity/smtCheckerTests/functions/constructor_hierarchy_diamond_3.sol

```diff
@@ -29,7 +29,6 @@ contract A is B2, B1 {
 // ====
 // SMTIgnoreCex: yes
 // ----
-// Warning 4984: (160-165): CHC: Overflow (resulting value larger than 2**256 - 1) happens here.
```

Comment:
I believe this is removed correctly. The trace of the counterexample was `B1.constructor(1)`, but `B1` cannot be deployed.